public/i18n/en.json

```diff
@@ -211,11 +211,13 @@
   "ftm-regional-spotlight-canada-header": "Regional Spotlight: #findthemasks Canada",
   "ftm-regional-spotlight-europe-header": "Regional Spotlight: #findthemasks Europe",
   "ftm-regional-spotlight-uk-header": "Regional Spotlight: #findthemasks UK",
+  "ftm-regional-spotlight-it-header": "Regional Spotlight: #findthemasks Italy",
   "ftm-regional-spotlight-india-header": "Regional Spotlight: #findthemasks India",
   "ftm-regional-spotlight-canada-text": "#findthemasks Canada was launched on March 29th, and now includes volunteers from across the country. The team is grateful for the help they have received from the #findthemasks team and its volunteers worldwide.<br /><br />See our Canadian partner organizations <a href='/ca/partners'>here</a>.",
   "ftm-regional-spotlight-india-text": "#findthemasks in India was launched on April 10th 2020, and now includes volunteers from across the country. The team is grateful for the help they have received from the #findthemasks team and its volunteers worldwide.<br /><br />See our Indian partner organizations <a href='/in/partners'>here</a>.",
   "ftm-regional-spotlight-europe-text": "#findthemasks Europe now includes volunteers from across the region. The team is grateful for the help they have received from the #findthemasks team and its volunteers worldwide.<br /><br />See our partner organizations <a href='/partners'>here</a>.",
   "ftm-regional-spotlight-uk-text": "#findthemasks UK now includes volunteers from across the region. The team is grateful for the help they have received from the #findthemasks team and its volunteers worldwide.<br /><br />See our partner organizations <a href='/partners'>here</a>.",
+  "ftm-regional-spotlight-it-text": "#findthemasks Italy is now looking for new members to join us. The team is grateful for the help they have received from the #findthemasks team and its volunteers worldwide.<br /><br />See our partner organizations <a href='/partners'>here</a>.",
```

Comment:
@NickB64  "join us" is missing the a href tag to link to the whoweare.html page :)